bundle/tests/quality_monitor/databricks.yml

```diff
@@ -24,14 +31,14 @@ targets:
       quality_monitors:
         my_monitor:
           table_name: "main.test.staging"
-          output_schema_name: "staging"
+          output_schema_name: "main.test"
```

Comment:
You can omit these overrides since they no longer override anything.

Reply:
Let me just change these values again to show how to differentiate them a bit.